### PR TITLE
TN-1737-fix-language-syntax

### PR DESCRIPTION
**Pull request by mcjustin**
... for both 14701 "CHU de Quebec - Universite Laval", and 14669 "Tygerberg Hospital"

### CODE REVIEW

File: portal/config/eproms/Organization.json [hunk 1]
```diff
@@ -1416,6 +1416,7 @@
           "timezone": "America/New_York",
           "url": "http://hl7.org/fhir/StructureDefinition/user-timezone",
           "valueCodeableConcept": {
+            "url": "http://hl7.org/fhir/valueset/languages",
```
Comment (achen2401) [inline portal/config/eproms/Organization.json]: @mcjustin Justin, shouldn't there be a bracket for the timezone - as it is a separate extension item, and I think system url shouldn't be part of the valueCodeableConcept dictionary item:
``{
         "timezone": "America/New_York",
          "url": "http://hl7.org/fhir/StructureDefinition/user-timezone"
    **}**, 
   {
         **"url": "http://hl7.org/fhir/valueset/languages"**,
         "valueCodeableConcept": {
         ......
}``

but correct me if I am wrong though, was just doing comparisons with others in the json

Comment (mcjustin) [reply portal/config/eproms/Organization.json]: @achen2401  can you please review again?

File: portal/config/eproms/Organization.json [hunk 2]
```diff
@@ -1414,8 +1414,11 @@
       "extension": [
         {
           "timezone": "America/New_York",
-          "url": "http://hl7.org/fhir/StructureDefinition/user-timezone",
+          "url": "http://hl7.org/fhir/StructureDefinition/user-timezone"
+        },
+		{
           "valueCodeableConcept": {
+            "url": "http://hl7.org/fhir/valueset/languages",
```
Comment (achen2401) [inline portal/config/eproms/Organization.json]: @mcjustin Justin, I think, the url item, ```http://hl7.org/fhir/valueset/languages```, is misplaced.  I think it should be present before valueCodeableConcept and after the beginning bracket, right?
```{ "url": "http://hl7.org/fhir/valueset/languages", "valueCodeableConcept": {...```

Comment (mcjustin) [reply portal/config/eproms/Organization.json]: sigh - please review again @achen2401

**Pull request merged**